catalog/app/website/pages/AwsMarketplace/Hero.tsx

```diff
@@ -0,0 +1,111 @@
+import cx from 'classnames'
+import * as React from 'react'
+import * as M from '@material-ui/core'
+
+import { useTalkToUs } from 'components/TalkToUs'
+import ChevronLink from 'website/components/ChevronLink'
+
+import logos from './logos.svg'
+
+const useStyles = M.makeStyles((t) => ({
+  root: {
+    maxWidth: '546px',
+  },
+  logos: {
+    background: `50% 50% url("${logos}") no-repeat`,
+    height: '99px',
+    marginBottom: t.spacing(6),
+  },
+  tagline: {
+    color: t.palette.text.secondary,
+    marginBottom: t.spacing(3),
+  },
+  buttons: {
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'center',
+    marginBottom: t.spacing(4),
+  },
+  button: {
+    minWidth: t.spacing(26),
+    '& + &': {
+      marginLeft: t.spacing(1),
+    },
+  },
+  buttonOutlined: {
```

Comment:
Hover behavior is beautiful, but this needs to be more prominent. Can it be semi-transparent?